src/app/static/src/js/components/runWorkflow/runWorkflowReport.vue

```diff
@@ -29,7 +29,33 @@
             </div>
             <div class="pb-4" id="workflow-reports">
                 <h3 id="report-sub-header">Reports</h3>
-                <div>
+                <div v-if="!importFromCsvIsEnabled" id="choose-import-from">
```

Comment:
Nearly there but this switch seems to be the wrong way around. This reads as "if import from csv is *not* enabled, show the component". 

To make this work as expected you want:

`const workFlowReport = !!urlParams.get('workFlowReport')`

and then you can remove the `!` on this line (and set the switch to true rather than false in the tests)